polaris.shopify.com/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM node:16
+
+ENV PORT 3000
+
+# Create app directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Installing dependencies
+COPY package*.json /usr/src/app/
+RUN npm install
+
+# Copying source files
+COPY . /usr/src/app
```

```suggestion
COPY . .
```

Same as above: https://github.com/Shopify/polaris/pull/5358/files#r836704378